Cinema system/Cinema system/Services/CinemaService.cs

```diff
@@ -0,0 +1,84 @@
+﻿using CinemaSystem.Interfaces;
+using CinemaSystem.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace CinemaSystem.Services
+{
+    public class CinemaService : ICinemaService
+    {
+
+        private readonly CinemaDBContext _db;
+
+        public CinemaService(CinemaDBContext db)
+        {
+            _db = db;
+        }
+
+        public bool IsCinemaExists(int cinemaId)
+        {
+            return _db.Cinemas.Any(c => (c.Id == cinemaId && !c.IsDeleted));
+        }
+
+        public bool IsCinemaExists(CinemaView cinema)
```

Comment:
Why do you need this method? I don't see any usages of it.

Reply:
I use this in controller method CreateCinema to determine if cinema with such data already exists.

Reply:
I use this in controller method CreateCinema to determine if cinema with such data already exists.